### PR TITLE
anonymize unused parameters

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: bin/genEvalSpecializations.py [hunk 1]
```diff
@@ -374,7 +374,7 @@ class Evaluation<ValueT, {{ numDerivs }}>
     }
 
     template <class RhsValueType>
-    static Evaluation createVariable(const Evaluation& x, const RhsValueType& value, int varPos)
```
Comment (andlaus) [inline bin/genEvalSpecializations.py]: like with OPM/ewoms#523, please silence such warnings using the OPM_UNUSED macros. (I will not mention this from here on, but please fix the other signatures as well.)

**Comment by akva2** [comment]
updated.

**Comment by akva2** [comment]
jenkins build this please

File: opm/material/thermal/NullSolidEnergyLaw.hpp [hunk 2]
```diff
@@ -50,7 +50,7 @@ class NullSolidEnergyLaw
      * This solid energy law simply returns 0.
      */
     template <class FluidState, class Evaluation = typename FluidState::Scalar>
-    static Evaluation solidInternalEnergy(const Params& params, const FluidState& fluidState)
+    static Evaluation solidInternalEnergy(const Params& params OPM_UNUSED, const FluidState&)
```
Comment (andlaus) [inline opm/material/thermal/NullSolidEnergyLaw.hpp]: that one was missed. I'll merge anyway if you want to avoid the jenkins dance...

Comment (akva2) [reply opm/material/thermal/NullSolidEnergyLaw.hpp]: i don't get what you mean.

Comment (andlaus) [reply opm/material/thermal/NullSolidEnergyLaw.hpp]: `fluidState` parameter removed instead of marked unused

Comment (akva2) [reply opm/material/thermal/NullSolidEnergyLaw.hpp]: sorry for being slow. updated.

**Comment by akva2** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
Comments addressed, all green, merging!

**Pull request merged**